assignments/arrays.js

```diff
@@ -63,34 +63,64 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+
+
+console.log(obinventor[32]);
+
+
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+  
+        for (let i = 0; i < inventory.length; i++){
+      if (inventory[i] === inventory[inventory.length - 1]){
+        let lastCar = inventory[i];
+
+        console.log(lastCar);
+      }
+  }
```

Comment:
again, read the instructions carefully as you only need to log the make and model of the last car, and here you're logging the entire object. good execution of a for loop, but you can actually forgo it at all in this case since the last car in the array can always be found and stored with something like this: ```let lastCar = inventory[inventory.length -1]``` and then you can reference the make and model like ```lastCar.car_make```